data/language/pl-PL.yml

```diff
@@ -2090,47 +2040,47 @@ strings:
   1995: '{SMALLFONT}{COLOUR BLACK}Rekordy prędkości'
   1996: Rekordy Prędkości
   1997: '{COLOUR WINDOW_2}Rekord prędkości usług na lądzie: {COLOUR BLACK}{VELOCITY}'
-  1998: '{COLOUR WINDOW_2}Rekord prędkości usługi powietrznej: {COLOUR BLACK}{VELOCITY}'
-  1999: '{COLOUR WINDOW_2}Rekord prędkości usługi wodnej: {COLOUR BLACK}{VELOCITY}'
+  1998: '{COLOUR WINDOW_2}Rekord prędkości usług w powietrzu: {COLOUR BLACK}{VELOCITY}'
+  1999: '{COLOUR WINDOW_2}Rekord prędkości usług na wodzie: {COLOUR BLACK}{VELOCITY}'
   2000: '{COLOUR BLACK}Osiągnięte przez {STRINGID} - {DATE DMY}'
   2001: Potwierdź zmianę trybu wyświetlania
   2002: '{COLOUR WINDOW_2}Rozdzielczość ekranu zmieniona na {UINT16 RAW} x {UINT16 RAW}'
   2003: '{COLOUR WINDOW_2}_'
   2004: '{COLOUR WINDOW_2}Nazwa pliku:'
   2005: "{COLOUR WINDOW_2}Folder: {COLOUR BLACK}{STRINGID}"
-  2006: '{SMALLFONT}{COLOUR BLACK}Przenieś jeden poziom do folderu nadrzędnego'
+  2006: '{SMALLFONT}{COLOUR BLACK}Przejdź do folderu nadrzędnego'
   2007: '{COLOUR WINDOW_2}Spółka: {COLOUR BLACK}{STRINGID}'
   2008: '{COLOUR WINDOW_2}Data: {COLOUR BLACK}{DATE DMY}'
   2009: '{COLOUR WINDOW_2}Postęp wyzwania: {COLOUR BLACK}{INT16}%'
   2010: '{COLOUR WINDOW_2}Wyzwanie: {COLOUR BLACK}Zakończono'
   2011: '{COLOUR WINDOW_2}Wyzwanie: {COLOUR BLACK}Porażka'
-  2012: 'Czy zastąpić istniejący plik: “{STRINGID}”?'
+  2012: 'Czy zastąpić istniejący plik: "{STRINGID}"?'
```

Comment:
Please take care to use `“` and `”` instead of `"`. They map to our sprite font.